### DISCUSSION TITLE
How to deploy teledrive to vercel

### DISCUSSION THREAD

**Discussion by churongcon** [mgilangjanuar/teledrive#231] [Q&A]
Hello
I want deploy teledrive to vercel for personal use, anyone can give me a tutorial?

I have fork this repo and deploy to vercel but it dont work
https://teledrive-eight.vercel.app/

**Comment by mgilangjanuar** [comment]
You should deploy with [vercel-cli](https://vercel.com/docs/cli) and run `npx vercel --token ${VERCEL_TOKEN} --prod`

Here's the flow that I used in GitHub actions: https://github.com/mgilangjanuar/teledrive/blob/staging/.github/workflows/build-staging.yml
